docs/rest.md

```diff
@@ -5,10 +5,10 @@ description: Call your REST APIs inside your GraphQL queries.
 
 Calling REST APIs from a GraphQL client opens the benefits GraphQL for more people, whether:
 
-* You are in a front-end developer team that wants to try GraphQL without asking for the backend team to implement a GraphQL server.
-* You have no access to change the backend because it's an existing set of APIs, potentially managed by a 3rd party.
-* You have an existing codebase, but you're looking to evaluate whether GraphQL can work for your needs.
-* You have a large codebase, and the GraphQL migration is happening on the backend, but you want to use GraphQL *now* without waiting!
+- You are in a front-end developer team that wants to try GraphQL without asking for the backend team to implement a GraphQL server.
```

Comment:
Sorry for the markdown churn. The prettier in my editor auto-formatted it. I can revert if need be.

Reply:
Please do revert, the linter settings we're under want the reverse styling. lol (my personal settings match yours)

Reply:
No problem! Will do.